GNUmakefile

```diff
@@ -66,7 +66,7 @@ define set_rpath
 endef
 
 hls: bindist/ghcs
-	for ghc in $(shell [ -e "bindist/ghcs-`uname`" ] && cat "bindist/ghcs-`uname`" || cat "bindist/ghcs") ; do \
+	for ghc in $(shell [ -e "bindist/ghcs-`uname -o`" ] && cat "bindist/ghcs-`uname -o`" || cat "bindist/ghcs") ; do \
```

Comment:
`-o` is not a posix option afais: https://pubs.opengroup.org/onlinepubs/009695399/utilities/uname.html

Reply:
I tried it on Windows and FreeBSD, and it seems like the only way to detect mingw without a version number so I think it is OK.